velox/dwio/type/fbhive/HiveTypeParser.cpp

```diff
@@ -50,7 +50,6 @@ HiveTypeParser::HiveTypeParser() {
   setupMetadata<TokenType::Short, TypeKind::SMALLINT>("smallint");
   setupMetadata<TokenType::Integer, TypeKind::INTEGER>({"integer", "int"});
   setupMetadata<TokenType::Long, TypeKind::BIGINT>("bigint");
-  setupMetadata<TokenType::Date, TypeKind::DATE>("date");
```

Comment:
Doesn't seem reasonable to leave TokenType::Date left out. Please can you double check this.

Reply:
Updated, thanks!